lib/ConnectionConfig.js

```diff
@@ -69,7 +69,7 @@ ConnectionConfig.mergeFlags = function mergeFlags(defaultFlags, userFlags) {
 
   // Merge the new flags
   for (var flag in newFlags) {
-    if (allFlags[flag] !== false) {
+    if (allFlags[flag] !== newFlags[flag]) {
```

Comment:
Is this really needed? The issue here is that this allows users to now enable flags this module disables because the flag is not actually supported by the module. If the flag is now supported by the module, you can just remove the blacklist of that flag instead of changing this part.

Reply:
This is from the previous PR -  @nmggithub?